sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql

```diff
@@ -62,6 +63,66 @@ desktop_activity_stream_events AS (
     `moz-fx-data-shared-prod.activity_stream.events`
   WHERE
     DATE(submission_timestamp) = @submission_date
+    AND CAST(metadata.user_agent.version AS INT64) < 123
+  GROUP BY
+    1,
+    2
```

Comment:
Making the group by columns explicit

```suggestion
    client_id,
    submission_date
```